tests/integration-tests/configs/common/common.yaml

```diff
@@ -540,6 +540,16 @@ storage:
         instances: {{ common.INSTANCES_DEFAULT_ARM }}
         oss: ["ubuntu2204"]
         schedulers: ["slurm"]
+  test_fsx_lustre.py::test_fsx_lustre_dra:
+    dimensions:
+      - regions: [ "eu-west-2" ]
+        instances: {{ common.INSTANCES_DEFAULT_X86 }}
+        oss: [ "alinux2" ]
+        schedulers: [ "slurm" ]
+      - regions: [ "eu-north-1" ]
+        instances: {{ common.INSTANCES_DEFAULT_ARM }}
+        oss: [ "ubuntu2204" ]
+        schedulers: [ "slurm" ]
```

Comment:
We can have the test run on a single region with a single os and instance type. Because this feature doesn't depends on os and instance type.